assets/shaders/compute_pass.comp

```diff
@@ -7,6 +7,7 @@
 
 layout(local_size_x = 16, local_size_y = 16) in;
 layout(binding = 0, rgba8) uniform writeonly image2D result_image;
+layout(binding = 7, rgba8) uniform image2D temporal_image;
```

Comment:
Why is this not in an order? seems odd to go from 0 to 7 to 1

Reply:
I personally believe it makes more sense to have the same types, closer together. (As this also reflects the shader variable order)

Reply:
Then just reorder the other bindings, thats much clearer.

Reply:
okay did it